src/test/calculate.test.js

```diff
@@ -0,0 +1,174 @@
+import calculate from '../logic/calculate';
```

Comment:
Your test did not cover lines 59,68,70,77,93 in `/logic/calculate`. It is either that the logic on those lines are useless or the test did not consider such case. Kindly investigate

Reply:
I'll investigate and make the necessary adjustments.